roles/commcare_analytics/defaults/main.yml

```diff
@@ -86,3 +86,11 @@ certbot_admin_email: admin@example.com
 certbot_domains:
   - example.com
   - www.example.com
+
+# Database backup variables
+postgresql_backup_dir: "/var/tmp"
+postgresql_backup_db_list: "commcare_analytics, superset"
+postgresql_user_home_dir: "/var/lib/postgresql"
+
+aws_region: us-west-2
```

Comment:
This might change. We still need to create a separate AWS account + bucket for this

Reply:
Putting AWS config in here does feel awkward, but I don't have a better suggestion. I'm curious what other people think.

Reply:
We can move the aws_region to the vault file also?

Reply:
If the vault file has other AWS config (and I guess it does, doesn't it?) then that sounds right.

Reply:
I don't think so. I haven't seen the vault file (yet). From the code, it looks like the only variable we read from that file is `vault_ansible_user`

Reply:
So apparently we need not the vault_ansible_user in the vault file. Not sure how it works, but my local testing was successful without it. I was able to construct a vault file that we can use.